doc/example/reference.cpp

```diff
@@ -507,14 +517,17 @@ namespace constraints_example_1
 //[ constraints_example_1
 MOCK_CLASS( mock_class )
 {
-    MOCK_METHOD( method, 2, void( int, const std::string& ) )
+    MOCK_METHOD( method1, 2, void( int, const std::string& ) )
+    MOCK_METHOD( method2, 2, void( int, const std::string& ) )
 };
 
 BOOST_AUTO_TEST_CASE( demonstrates_adding_builtin_constraints )
 {
    mock_class c;
-   MOCK_EXPECT( c.method ).with( mock::equal( 3 ), mock::equal( "some string" ) );
-   MOCK_EXPECT( c.method ).with( 3, "some string" );                               // equivalent to the previous one using short-cuts
+   MOCK_EXPECT( c.method1 ).with( mock::equal( 3 ), mock::equal( "some string" ) );
+   MOCK_EXPECT( c.method2 ).with( 3, "some string" );                               // equivalent to the previous one using short-cuts
```

Comment:
It’s not exactly equivalent anymore now if it’s two different methods.

Reply:
Adapted the comment, better?

Reply:
OK.